### ISSUE TITLE
FPP Crashing and then Restarting

### ISSUE THREAD

**Issue by garylred** [FalconChristmas/fpp#929]
Been having this issue for about a week. My show will be running and then the K32A this on my megatree will freeze. I have upgraded to FPP version 4.6 hoping there was a bug fix that would resolve the issue but it did not. I have attached the Fpdd log so that you can see the crash message. Really need to know what might be causing this as it is the only controller in my show that is having this issue.
[fppd.log](https://github.com/FalconChristmas/fpp/files/5702940/fppd.log)

**Comment by jfeger77** [comment]
Im not an expert, but it looks like you have channel logging enabled.  Set your logging back to INFO / MOST.  This is a known issue I believe (recently).

**Comment by garylred** [comment]
It's funny you say that. I looked at the logging on that controller and channel logging was the only thing enabled. I never modified the logging level. I looked at my other K32 and it was set at default and it has not had any issues. I changed the logging level back to default on the K32 for my tree and I will keep an eye on it. I hope that fixes it. Thank you for the suggestion as sometime you just need another set of eyes to look at it.
